### PR TITLE
Fix some of the german translation.

### PR DESCRIPTION
**Pull request by redstonekasi**
Changed some of the translations to make more sense.

### CODE REVIEW

**Comment by Sturmlilie** [comment]
Small note: If you wanna be consistent with vanilla, use "Objekte" for "entities". [1]

[1] https://minecraft-de.gamepedia.com/Objekt

File: src/main/resources/assets/lambdynlights/lang/de_de.json [hunk 1]
```diff
@@ -4,16 +4,16 @@
   "lambdynlights.menu.canvas.1": "Mit Canvas ist es empfohlen, die dynamische Entitäten-Beleuchtung auszuschalten!",
   "lambdynlights.menu.canvas.2": "Dynamische Beleuchtung mit Canvas ist zur Zeit sehr Leistungsbedürftig!",
   "lambdynlights.option.block_entities": "Blockentitäten",
```
Comment (AlexanderRitter02) [inline src/main/resources/assets/lambdynlights/lang/de_de.json]:
```suggestion
  "lambdynlights.option.block_entities": "Blockobjekte",
```
This is the German term used in the Minecraft wiki, for consistency: https://minecraft-de.gamepedia.com/Blockobjektdaten

File: src/main/resources/assets/lambdynlights/lang/de_de.json [hunk 2]
```diff
@@ -4,16 +4,16 @@
   "lambdynlights.menu.canvas.1": "Mit Canvas ist es empfohlen, die dynamische Entitäten-Beleuchtung auszuschalten!",
   "lambdynlights.menu.canvas.2": "Dynamische Beleuchtung mit Canvas ist zur Zeit sehr Leistungsbedürftig!",
```
Comment (AlexanderRitter02) [inline src/main/resources/assets/lambdynlights/lang/de_de.json]:
```suggestion
  "lambdynlights.menu.canvas.2": "Dynamische Beleuchtung mit Canvas ist zur Zeit sehr leistungsbedürftig!",
```
Small spelling correction.

File: src/main/resources/assets/lambdynlights/lang/de_de.json [hunk 3]
```diff
@@ -4,16 +4,16 @@
...
   "lambdynlights.option.block_entities": "Blockentitäten",
-  "lambdynlights.option.entities": "Entitäten",
+  "lambdynlights.option.entities": "Objekte",
   "lambdynlights.option.mode": "Dynamische Lichter",
-  "lambdynlights.option.water_sensitive": "Wasser-sensibel Prüfung",
+  "lambdynlights.option.water_sensitive": "Wasserempfindlichkeit",
   "lambdynlights.toast.first_time": "Automatisch ausgeschaltet, prüfen Sie die Mod- oder Videoeinstellungen nach!",
```
Comment (AlexanderRitter02) [inline src/main/resources/assets/lambdynlights/lang/de_de.json]: 1. That must either be:
`"prüfen Sie **in** den Mod- oder Videoeinstellungen nach!"`
or
`"prüfen Sie die Mod- oder Videoeinstellungen!"`
A mix doesn't make sense gramatically.

2. Default **is not** equal to automatic, it is rather `"Standardmäßig"`.

File: src/main/resources/assets/lambdynlights/lang/de_de.json [hunk 4]
```diff
@@ -4,16 +4,16 @@
...
   "lambdynlights.tooltip.block_entities": "Schaltet Blockentitäten dynamische Beleuchtung ein.",
```
Comment (AlexanderRitter02) [inline src/main/resources/assets/lambdynlights/lang/de_de.json]:
```suggestion
  "lambdynlights.tooltip.block_entities": "Schaltet dynamische Beleuchtung für Blockobjekte ein.",
```
Same as above, term used in Minecraft wiki.
Also correct the word order in the sentence.

File: src/main/resources/assets/lambdynlights/lang/de_de.json [hunk 5]
```diff
@@ -4,16 +4,16 @@
...
-  "lambdynlights.tooltip.creeper_lighting": "Ändert das Verfahren der dynamische Beleuchtung des Creepers\n- %s schaltet dieses aus.\n- %s: konstante Beleuchtung durch explosierende Creepers.\n- %s: steigende Beleuchtung durch explosierende Creepers.",
-  "lambdynlights.tooltip.entities": "Aktiviert die dynamische Beleuchtung von Entitäten. (Spielerbeleuchtung ist immer aktiviert)",
+  "lambdynlights.tooltip.creeper_lighting": "Ändert das Verhalten der dynamische Beleuchtung des Creepers\n- %s schaltet dieses aus.\n- %s: konstante Beleuchtung durch explosierende Creepers.\n- %s: steigende Beleuchtung durch explosierende Creepers.",
```
Comment (AlexanderRitter02) [inline src/main/resources/assets/lambdynlights/lang/de_de.json]:
```suggestion
  "lambdynlights.tooltip.creeper_lighting": "Ändert das Verhalten der dynamische Beleuchtung bei Creepern\n- %s schaltet dieses aus.\n- %s: konstante Beleuchtung durch explodierende Creepers.\n- %s: Zunehmende Beleuchtung durch explodierende Creepers.",
```

1. _"des Creepers" -> "bei Creepern"_. The sentence applies to all creepers, not only one. Also sounds better.
2. "Explodieren" is written with a **"d"**: https://www.duden.de/suchen/dudenonline/explodierende

File: src/main/resources/assets/lambdynlights/lang/de_de.json [hunk 6]
```diff
@@ -4,16 +4,16 @@
...
+  "lambdynlights.tooltip.entities": "Aktiviert die dynamische Beleuchtung von Objekten. (Spielerbeleuchtung ist immer aktiviert)",
   "lambdynlights.tooltip.mode.1": "Aktiviert dynamische Beleuchtung. Wenn es aktiviert ist wird zB. einem Fackel-haltendem Spieler seine Umgebung beleuchtet.",
```
Comment (AlexanderRitter02) [inline src/main/resources/assets/lambdynlights/lang/de_de.json]:
```suggestion
  "lambdynlights.tooltip.mode.1": "Aktiviert dynamische Beleuchtung. Wenn aktiv, wird z. B. ein Spieler, der eine Fackel hält, die Umgebung beleuchten."
```
The current translation is a bit rough, improve it and also correct punctuation (comma).

File: src/main/resources/assets/lambdynlights/lang/de_de.json [hunk 7]
```diff
@@ -4,16 +4,16 @@
...
   "lambdynlights.tooltip.mode.2": "%s und %s haben eine kleine Verzögerung, aber lösen weniger Lichtupdates aus.",
   "lambdynlights.tooltip.mode.3": "Mit %s sind die Bewegungen von Lichtquellen flüssig.",
-  "lambdynlights.tooltip.tnt_lighting": "Ändert das Verfahren der dynamischer Beleuchtung von TNT.\n- %s schaltet dieses aus.\n- %s: konstante Beleuchtung.\n- %s steigende Beuleuchtung.",
-  "lambdynlights.tooltip.water_sensitive": "Aktiviert die Wasser-sensibel Prüfung: einige Objekten werden keinen Licht ins Wasser abgeben."
+  "lambdynlights.tooltip.tnt_lighting": "Ändert das Verhalten der dynamischer Beleuchtung von TNT.\n- %s schaltet dieses aus.\n- %s: konstante Beleuchtung.\n- %s steigende Beuleuchtung.",
+  "lambdynlights.tooltip.water_sensitive": "Aktiviert die Wasserempfindlichkeit: einige Gegenstände werden kein Licht im Wasser abgeben."
```
Comment (AlexanderRitter02) [inline src/main/resources/assets/lambdynlights/lang/de_de.json]:
```suggestion
  "lambdynlights.tooltip.water_sensitive": "Aktiviert die Wasserempfindlichkeit: Einige Gegenstände werden kein Licht im Wasser abgeben."
```
A new sentence after a colon should be capitalized: https://www.duden.de/sprachwissen/rechtschreibregeln/doppelpunkt

**Review by AlexanderRitter02 — CHANGES_REQUESTED** [review]

**Pull request merged**